src/components/Searchbar.js

```diff
@@ -34,26 +43,34 @@ const Searchbar = () => {
         setQuery(event.target.value)
     }
 
+    const handleClickSong = event => {
+        const songItem = event.target.getAttribute("data-index");
+        setSelectedSong({
+            title_short: results[songItem].title_short,
+            preview: results[songItem].preview,
+            artist: results[songItem].artist.name,
+            deezer_id: results[songItem].id
+        });
```

Comment:
Nice job keeping it simple here with the array[index] instead of using a mapped filter or something. Readabile. 🧾🕶